plugins/placeholder_select/plugin.js

```diff
@@ -12,59 +12,60 @@
 
 CKEDITOR.plugins.add('placeholder_select',
   {
-    requires : ['richcombo'],
-    init : function( editor )
-	{
-		//  array of placeholders to choose from that'll be inserted into the editor
+    requires: ['richcombo'],
+    init: function (editor) {
+      //  array of placeholders to choose from that'll be inserted into the editor
       var placeholders = [];
 
-		// init the default config - empty placeholders
+      // instances
+      var editorInstance = CKEDITOR.instances["problem-description"];
```

Comment:
"problem-description" is the default instanceId given to Ckeditor instance. In case of iNvite screen, it can be possible that it will work because we might be using default instanceId. But should we make it dynamic in case if it is required for other use-cases ?